### PR TITLE
Hides the vlan attribute for a VxLAN port.

### PR DESCRIPTION
**Pull request by tomohikokimura**
The Vlan ID is not part of the original bridge port attributes. The
current CLI shows "VLAN: -1" for a VxLAN port when we list ports for a
bridge with a VxLAN port, which is useless and misleading. This patch
hides the Vlan attribute for a VxLAN port.

**Old CLI output**
midonet> bridge bridge5 list port
port port0 device bridge5 state up vlan -1 management-ip 119.15.112.22 vni 12304763

**New CLI output**
midonet> bridge bridge5 list port
port port0 device bridge5 state up management-ip 119.15.112.22 vni 12304763

Fix MN-2386

### CODE REVIEW

**Comment by tomohikokimura** [comment]
Hello Brandon.

This is a trivial bug fix. Please take a look.

Best,
Tomohiko

**Comment by tomohikokimura** [comment]
@srvaroa @brandon-berg I updated the patch. Please take another look. Thanks!

File: src/midonetclient/port.py [hunk 1]
```diff
@@ -30,6 +30,7 @@
 class Port(resource_base.ResourceBase, AdminStateUpMixin):
 
     media_type = vendor_media_type.APPLICATION_PORT_V2_JSON
+    vxlan_port_type = 'Vxlan'
```
Comment (brandon-berg) [inline src/midonetclient/port.py]: I'm not really familiar with Python conventions. Should this be in all-caps?

**Comment by brandon-berg** [comment]
Looks okay other than that, though. Approving conditioned on your dealing with the capitalization issue if you think it's appropriate to do so.

**Comment by srvaroa** [comment]
Apart of Brandon's comment, I'd suggest to amend the commit message which is misleading now. Should specify only for vxlan port.

**Comment by srvaroa** [comment]
Oh ignore me, the commit is ok, it's the pull request that keeps the old name.

**Comment by tomohikokimura** [comment]
Thank you for your review, @brandon-berg and @srvaroa . I newly introduced a global variable PORT_TYPE_VXLAN defined inside port.py. I also updated the pull request title / message. I'm merging this for now.

**Comment by srvaroa** [comment]
Thanks @tomohikokimura

**Pull request merged**